api/HTMLTextAreaElement.json

```diff
@@ -191,45 +190,138 @@
           }
         }
       },
-      "reportValidity": {
+      "cols": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLTextAreaElement/reportValidity",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": "40"
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "≤6"
+            },
+            "opera": {
+              "version_added": "≤12.1"
+            },
+            "opera_android": {
+              "version_added": "≤12.1"
+            },
+            "safari": {
+              "version_added": "≤4"
+            },
+            "safari_ios": {
+              "version_added": "≤3"
+            },
+            "samsunginternet_android": {
+              "version_added": "1.0"
+            },
+            "webview_android": {
+              "version_added": "1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "defaultValue": {
```

Comment:
I checked that there's no "defaultValue" entry of some other capitalization or on a parent interface.